skipchain/skipchain.go

```diff
@@ -371,12 +359,15 @@ func (s *Service) StoreSkipBlock(psbd *StoreSkipBlock) (*StoreSkipBlockReply, er
 			// Requesting creation of secondary forward link.
 			log.Lvlf2("%s: sending request for height %d to %s", s.ServerIdentity(),
 				i+1, back.Roster.List[0])
-			s.SendRaw(back.Roster.List[0], &ForwardSignature{
+			err := s.SendRaw(back.Roster.List[0], &ForwardSignature{
 				TargetHeight: i + 1,
 				Previous:     back.Hash,
 				Newest:       prop.Copy(),
 			})
-			// time.Sleep(time.Second)
+
+			if err != nil {
+				log.Lvlf2("%v", err)
```

Comment:
It's worth to put a warning here.

```suggestion
				log.Warn("%v", err)
```

Reply:
I changed it and also removed the unnecessary formatting